src/ui/public/kfetch/kfetch.ts

```diff
@@ -28,7 +28,9 @@ import { metadata } from '../metadata';
 class FetchError extends Error {
   constructor(public readonly res: Response, public readonly body?: any) {
     super(res.statusText);
-    Error.captureStackTrace(this, FetchError);
+    if (Error.captureStackTrace) {
```

Comment:
You could be _really_ thorough here and check if it's a function too. How about adding a comment too, since I didn't realize `captureStackTrace` was specific to V8?

```js
// captureStackTrace is only available in the V8 engine, so any browser using
// a different engine won't have access to this method.
if (typeof Error.captureStackTrace === 'function') {
  Error.captureStackTrace(this, FetchError);
}
```